### ISSUE TITLE
Error after minimizing the app.

### ISSUE THREAD

**Issue by jacynkaa** [mini2Dx/mini2Dx#86]
After minimizing the app I get error:
```
Exception in thread "LWJGL Application" java.lang.ClassCastException: java.base/[Ljava.lang.Object; cannot be cast to [Lcom.badlogic.gdx.LifecycleListener;
	at com.badlogic.gdx.backends.lwjgl.DesktopMini2DxGame.executeGame(DesktopMini2DxGame.java:167)
	at com.badlogic.gdx.backends.lwjgl.DesktopMini2DxGame$1.run(DesktopMini2DxGame.java:118)
```
and app crashes.

I use:
mini2DxVersion = '1.9.0'
gdxVersion = '1.9.8'
But with mini2DxVersion = '1.8.0' everything seems to be OK.

**Comment by jacynkaa** [comment]
PS. I "solved" this problem setting DesktopMini2DxConfig.pauseWhenMinimized nad DesktopMini2DxConfig.pauseWhenBackground to false, so isMinimized at DesktopMini2DxGame.java:164 is always false.

**Comment by tomcashman** [comment]
I've fixed this in https://github.com/mini2Dx/mini2Dx-ve/commit/38d8353dea523f020707cdf9817e5765bdecd56b and published 1.9.2 to Maven Central :)
